fetch-number-of-jobs-from-all-connected-controllers.groovy

```diff
@@ -0,0 +1,31 @@
+import com.cloudbees.opscenter.server.clusterops.steps.MasterGroovyClusterOpStep
+
+def retour = ''
+def stream = new ByteArrayOutputStream();
+def listener = new StreamBuildListener(stream);
+
+Jenkins.instance.getAllItems(com.cloudbees.opscenter.server.model.ConnectedMaster).each{
+    it.channel?.call(new MasterGroovyClusterOpStep.Script("""
+        class ComposedItem {
+            String controllerName
+            Integer numberOfJobs
+            String allJobs
+        }
+        
+        jobsController = [];
+
+        jenkins.model.Jenkins.get().allItems(hudson.model.Job).each {
+        	jobsController.push(it)
+        }
+
+        def aux = new ComposedItem();
+        aux.controllerName = "${it.name}"
+        aux.numberOfJobs = jobsController.size()
+        aux.allJobs = jobsController
```

Comment:
You have `allJobs` as a `String` in the `ComposedItem` but are passing a `ArrayList<Job>` (`jobsController = [];`)

Do you really want to pass the entire job object, or maybe just the name.

Reply:
In the end, it "does not mind" what type of Object are the attributes of ComposedItem because the internal script returns aux.dump() so in the end, is going to be stringified. But effectively, as the purpose is retrieving the names of the Jobs, I have modified the script to get `it.getFullName()` which is much cleaner.

I have updated the example output in my previous comment.